<launch>
  <!-- Iniciar o nÃ³ hw_interface e o de i2c -->
  <node name="hw_interface" pkg="robot_control" type="hw_interface" output="screen" />
  <node name="i2c_master" pkg="robot_control" type="i2c_master.py" output="screen" />
</launch>
